roxie/ccd/ccdmain.cpp

```diff
@@ -672,7 +672,16 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
         topology = loadConfiguration(useOldTopology ? nullptr : defaultYaml, argv, "roxie", "ROXIE", topologyFile, nullptr, "@netAddress");
         saveTopology();
         if (topology->getPropBool("expert/@profileStartup", false))
+        {
+            if (topology->hasProp("expert/@profileStartupInterval"))
+            {
+                double interval = atof(topology->queryProp("expert/@profileStartupInterval"));
```

Comment:
getPropReal(..)
is more concise

Reply:
I just assumed there wasn't such a thing! Will change.